### PR TITLE
Page objects and test

### PR DESCRIPTION
**Pull request by MOsinloye**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -4,4 +4,6 @@ node_modules
 dist/
 mochawesome-report/
 .vscode/
-yarn-error.log
\ No newline at end of file
+yarn-error.log
+output
+steps.d.ts
```
Comment (vianvi) [inline .gitignore]: I didn't see any steps.d.ts file , why do we need to ignore?

File: codecept.conf.js [hunk 2]
```diff
@@ -0,0 +1,34 @@
+exports.config = {
+  tests: "tests/**/*.js",
+  timeout: 10000,
+  output: "./output",
+  helpers: {
+    Puppeteer: {
+      url: "https://www-ccd.nonprod.platform.hmcts.net/list/case?jurisdiction=DIVORCE&case-type=DIVORCE&case-state=Submitted",
```
Comment (vianvi) [inline codecept.conf.js]: Please move urls and credentials to separate file

File: package.json [hunk 3]
```diff
@@ -1,16 +1,18 @@
...
+  "name": "div-ccd-e2e-tests",
   "version": "0.0.0",
   "private": true,
+  "description": "",
   "scripts": {
-    // "test-e2e": "NODE_PATH=. codeceptjs run  -c test/end2end --steps "
-  },
-  "dependencies": {
-    "cross-env": "^5.1.3"
+    "pretest": "npm run create:type-definitions",
```
Comment (vianvi) [inline package.json]: Not sure why we need pretest and npm.

File: .gitignore (hunk 1 above)
Comment (MOsinloye) [reply .gitignore]: The type definition file is to allow for intellisense.  Autocompletion as codecept puts it.
It's just to make writing  tests less tricky for me.
This is how Codecept advised to get code completion.

File: codecept.conf.js (hunk 2 above)
Comment (MOsinloye) [reply codecept.conf.js]: ok

File: package.json (hunk 3 above)
Comment (MOsinloye) [reply package.json]: I added this just to keep definition up to date. The definition file is gitignored anyway.
Again this is to help me with writing tests

File: codecept.conf.js (hunk 2 above)
Comment (MOsinloye) [reply codecept.conf.js]: Tests now depend on a TEST_ENV variable.

**Pull request merged**